app/code/Magento/Customer/Block/Adminhtml/Group/Edit/Form.php

```diff
@@ -1,148 +0,0 @@
-<?php
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
-namespace Magento\Customer\Block\Adminhtml\Group\Edit;
-
-use Magento\Customer\Controller\RegistryConstants;
-
-/**
- * Adminhtml customer groups edit form
- */
-class Form extends \Magento\Backend\Block\Widget\Form\Generic
```

Comment:
Please keep this block and mark as deprecated

Reply:
done